packages/workers-playground/src/QuickEditor/PreviewTab/useRefreshableIframe.tsx

```diff
@@ -69,6 +70,7 @@ export function useRefreshableIframe(
 		if (src) {
 			setUrl(src);
 		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this one is a tricky one to fix, excluding it for now

Reply:
same question here